lms/envs/common.py

```diff
@@ -2050,6 +2055,9 @@
 
     # Needed whether or not enabled, due to migrations
     'badges',
+
+    # Enables default site and redirects
+    'django_sites_extensions',
```

Comment:
Well, it is dependent on Django (redirects is a part of the core Django package). I think the naming of the package calls this out pretty well.